plugins/catalog-backend/src/ingestion/LocationReaders.ts

```diff
@@ -185,6 +190,7 @@ export class LocationReaders implements LocationReader {
             current,
             item.location,
             emit,
+            originLocation,
```

Comment:
at some time in the future it would be nice to revamp this api ... these arguments should have all been fields in a single object from the start, or maybe the locations should have been in a `context` object or something